srcpkgs/easystroke/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'easystroke'
+pkgname=easystroke
+version=0.6.0
+revision=1
+archs="i686 x86_64"
+wrksrc=patched-easystroke-master
+build_wrksrc=easystroke
+build_style=gnu-makefile
+hostmakedepends="pkg-config"
+makedepends="boost-devel dbus-glib-devel gtkmm-devel xorg-server-devel"
+short_desc="Gesture recognition application for X11"
+maintainer="lopid <github23429@rainslide.net>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/thjaeger/easystroke/wiki/"
+distfiles="http://openartisthq.org/easystroke/patched-easystroke-master.tar.bz2"
```

Comment:
We don't want packages using unversioned tarballs. This means any update from upstream breaks our build. Where did you get the version number from?